environments/staging/aws-resources.yml

```diff
@@ -35,8 +35,8 @@ pgproxy2-staging.instance_id: i-0638ff6218ed9f753
 pillow3-staging: 10.201.10.68
 pillow3-staging.instance_id: i-0ef555fec7fe64804
 proxy1-staging: 10.201.20.208
-proxy1-staging.instance_id: i-02e1fe640296f8db9
-proxy1-staging.public_ip: 3.89.61.176
+proxy3-staging: 10.201.20.47
+proxy3-staging.instance_id: i-0a53d5199910fc2b7
```

Comment:
Do we need to include `proxy3-staging.public_ip`?

Reply:
No , I have re setup in private subnet